docs/swagger.yaml

```diff
@@ -4841,6 +4841,9 @@ definitions:
       project_creation_restriction:
         type: string
         description: This attribute restricts what users have the permission to create project.  It can be "everyone" or "adminonly".
+      quota_per_project_enable:
```

Comment:
I'd prefer that we use a separate API to control the disable and enable quota, and we can add logic in the scenario from disable to enable.

But, it can be stored in the table of properties.